Common/VCL.Styles.Utils.inc

```diff
@@ -1,33 +1,34 @@
-//**************************************************************************************************
-//
-// Vcl.Styles.Utils.inc
-// file for the VCL Styles Utils
-// https://github.com/RRUZ/vcl-styles-utils/
-//
-// The contents of this file are subject to the Mozilla Public License Version 1.1 (the "License");
-// you may not use this file except in compliance with the License. You may obtain a copy of the
-// License at http://www.mozilla.org/MPL/
-//
-// Software distributed under the License is distributed on an "AS IS" basis, WITHOUT WARRANTY OF
-// ANY KIND, either express or implied. See the License for the specific language governing rights
-// and limitations under the License.
-//
-//
-// Portions created by Mahdi Safsafi [SMP3]   e-mail SMP@LIVE.FR
-// Portions created by Rodrigo Ruz V. are Copyright (C) 2013-2017 Rodrigo Ruz V.
-// All Rights Reserved.
-//
-//************************************************************************************************
-
-{$DEFINE USE_Vcl.Styles.Hooks}
-{$DEFINE HOOK_UXTHEME}
-{$DEFINE HOOK_TDateTimePicker}
-{$DEFINE HOOK_TProgressBar}
-{.$DEFINE LimitStylesToMainApplicationThread}
-
-{ Feature toggles for supported UxThemes - see Vcl.Styles.UxTheme }
-
-{$DEFINE HOOK_Button}
+//**************************************************************************************************
+//
+// Vcl.Styles.Utils.inc
+// file for the VCL Styles Utils
+// https://github.com/RRUZ/vcl-styles-utils/
+//
+// The contents of this file are subject to the Mozilla Public License Version 1.1 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.mozilla.org/MPL/
+//
+// Software distributed under the License is distributed on an "AS IS" basis, WITHOUT WARRANTY OF
+// ANY KIND, either express or implied. See the License for the specific language governing rights
+// and limitations under the License.
+//
+//
+// Portions created by Mahdi Safsafi [SMP3]   e-mail SMP@LIVE.FR
+// Portions created by Rodrigo Ruz V. are Copyright (C) 2013-2020 Rodrigo Ruz V.
+// All Rights Reserved.
+//
+//************************************************************************************************
+
+{$DEFINE USE_Vcl.Styles.Hooks}
+{$DEFINE HOOK_UXTHEME}
+{$DEFINE HOOK_TDateTimePicker}
+{$DEFINE HOOK_TProgressBar}
+{.$DEFINE LimitStylesToCurrentProcess}
```

Comment:
Please, keep the new "LimitStylesToMainApplicationThread" name in this DEFINE. 
"LimitStylesToCurrentProcess" was the old name which was misleading.

Reply:
Done. Thanks!